### ISSUE TITLE
Getting Entity not available: sensor.washer_remianing_time

### ISSUE THREAD

**Issue by flyingdutchman7588** [ShunichiSan/HomeAssistantApplianceCard#3]
I am getting this for the washer and dryer portion of it. Also getting another Entity not available at the bottom. I have the same LG washer and Dryer so I just removed the dishwahser. 

![image](https://user-images.githubusercontent.com/37387465/125676385-4f6e364d-1082-4d79-8ebf-f50328da2399.png)

I still have to upload the images to my local files so thats why the washer and dryer pics are not there and the card looks messed up. I have already installed: 
SmartThinQ LGE Sensors
vertical-stack-in-card
circle-sensor-card
text-element

I have also created a sensors.yaml in my config and copied the sensors.yaml code you had on your site. What am I doing wrong?

**Comment by kshoichi** [comment]
Check the following:

1. Looks like you have a typo in the name: sensor.washer_remianing_time. 
2. Make sure your configuration.yaml has a line that includes the sensors.yaml file. It should have a line that says: "sensor: !include sensors.yaml"
3. Check all the names in your card's yaml and also in the sensors.yaml to make sure they match what the SmartThinQ integration creates.

**Comment by flyingdutchman7588** [comment]
Thanks for getting back to me! 

1. The typo was just in the title of this thread. It is spelt correctly in HA. 
2. I didnt have that before so I added that now. It is just all in one line, right?
3. I checked the names in the sensors.yaml file and I think I may have found the issue but not sure. Below are all the entities from the washer
![image](https://user-images.githubusercontent.com/37387465/125686918-09c5e1af-2b51-4ee2-90a0-08a87cf5de77.png)
![image](https://user-images.githubusercontent.com/37387465/125686950-8f25114b-f4e5-488f-9533-45bee449172f.png)

So in your sensors.yaml, I see "states.sensor.washer.attributes.initial_time.split", "states.sensor.washer.attributes.remain_time.split" and "states.sensor.washer.attributes.remain_time.state" but I dont have that. Does that mean this won't work for me?

**Comment by kshoichi** [comment]
Yes, add that as its own line in your configuration.yaml then save and reboot. That line tells home assistant where to find your custom sensor data. Also note that if you configure any other sensors in the configuration.yaml you will need to move that information to the sensors.yaml file instead.
Regarding "states.sensor.washer.attributes.initial_time.split... the "sensor.washer" entity has many attributes and "remain_time" is one of them. So look for a "sensor.washer" entity and you'll see a list of attributes in the last column.

**Comment by flyingdutchman7588** [comment]
Ahh yes now I see the sensor.washer entity and its attributes. 

I restarted Home assistant and now I can see the remaining time up on the top but the bottom still has an error and when I hover over it, I cannot see what the error is. 

![image](https://user-images.githubusercontent.com/37387465/125689990-5be60acc-ebd0-4270-8b57-5c05dbda7a16.png)

I can click the circle which shows washer progress but there is still that error. Is it because I dont have the .png files saved in the folder and so it cannot find the images? I am at work so I have to wait till I go home if I want to access the www directory.

**Comment by flyingdutchman7588** [comment]
Ok, got another update...

So I uploaded the images and finally got a name for the error...Entity not available: sensor.washer_state. How do I proceed from this?

![image](https://user-images.githubusercontent.com/37387465/125698712-d0ee5eb1-43b2-48a7-9e5b-0011b6852022.png)

**Comment by kshoichi** [comment]
Your version of the integration might have a different name for that sensor. It might be the sensor_washer_run_state or sensor.washer_current_course from your image above. Best thing to do is to run a washer cycle and see which entity in Home Assistant shows what cycle it is running. Then you would change this by editing your Lovelace card. It also looks like you might need to also adjust the placement of your grey circles since everyone's system is a little different.

**Comment by flyingdutchman7588** [comment]
Thats a great idea! I will do that today and look to see what is the issue. It looks like I am almost there and I am super excited because this is an amazing card. Kudos to you for doing this and sharing it with us. 

For the grey circle, I know how to move it back and forth but how do I make the grey circle smaller? It looks like the circle is too big for the other circle.

**Comment by flyingdutchman7588** [comment]
Hi ShunichiSan, 

I have one more question. Looks like I ran the washer and dryer circle and found the correct settings. Now the issue I am having is more layout related. The grey circle in the center of the washing machine is on different areas of the page depending on how I look at my HA. 

When I edit the dashboard, this is what it looks like in the preview: 
![image](https://user-images.githubusercontent.com/37387465/126203748-7c1ae489-6030-404a-b41a-79e64f7b01c8.png)

However, when I click save, this is what it shows: 
![image](https://user-images.githubusercontent.com/37387465/126203837-00226ab4-0367-4702-abfb-691ba0f3b204.png)

Then on my Android phone, it looks like this:
![image](https://user-images.githubusercontent.com/37387465/126204039-0738a209-f727-4ced-8337-3c07083ca59c.png)


On my iPad it also looks close to what it looks on my Android. Is there any way to "lock" the circle?

**Comment by drezin2112** [comment]
I am  having the same issue where the washer is calling an unavailable attribute, but not sure how to fix it would you please help on how and what to add to the confog file?
![image](https://user-images.githubusercontent.com/89281275/161183690-96384d47-cf71-4e64-9530-062ed0a6a8ae.png)

![image](https://user-images.githubusercontent.com/89281275/161183747-73c44e52-9e0f-486c-b92d-c8cbc0fb863a.png)

**Comment by kshoichi** [comment]
Go into developer tools and find the sensors that were created by the LG integration. They might be named differently than mine. Once you find the missing entity in developer tools, put that in your card's yaml. It's also possible that your appliance doesn't report the missing data.
